spec/support/shared_examples/api_create_declaration_endpoint_support.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+RSpec.shared_examples "an API create declaration endpoint" do
```

Comment:
Can we make this a generic "an API create endpoint" that just checks it accepts a POST request of the correct body and calls the expected service/serializer? Similar to [this](https://github.com/DFE-Digital/npq-registration/blob/f6ab9971671edf56485560ffc181ca6a43981e92/spec/support/shared_examples/api_update_endpoint_support.rb)